core/kazoo_documents/src/kzd_voicemail_box.erl

```diff
@@ -17,6 +17,8 @@
         ,pin/1, pin/2
         ,mailbox_number/1, mailbox_number/2
         ,pin_required/1, pin_required/2
+        ,is_ff_rw_enabled/1, is_ff_rw_enabled/2
```

Comment:
Should not duplicate this between `kzd_vmboxes` and `kzd_voicemail_boxes`. Please focus on `kzd_vmboxes` as this is the module that tracks the schema.

Reply:
@jamesaimonetti I believe we have resolved this and the other tickie items.  Can you take a look?